frontend/sw360-portlet/src/main/webapp/js/component/autocomplete.js

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
This is an example for wrapping old code into a module.
Of course the old code will remain in the global namespace but this way every user can be migrated to use the module instead.
If all users of the module are migrated, the global function can be moved into the module as well.